src/lib/Makefile

```diff
@@ -80,20 +102,7 @@ distclean: clean
 libHYPRE.a: ${FILES_HYPRE}
 	@echo  "Building libHYPRE ... "
 	rm -f $@
-	${AR} $@ $(FEIHYPREFILES) $(FEMLIFILES) $(IJMVFILES)
-	${AR} $@ $(EUCLIDFILES) $(PARASAILSFILES) $(PILUTFILES) $(DISTMATRIXFILES)
-	${AR} $@ $(KRYLOVFILES)
-	${AR} $@ $(MATMATFILES)
-	${AR} $@ $(MULTIVECFILES)
-	${AR} $@ $(PARCSRLSFILES)
-	${AR} $@ $(PARCSRMVFILES)
-	${AR} $@ $(PARCSRBLOCKMVFILES)
-	${AR} $@ $(SEQMVFILES)
-	${AR} $@ $(SSTRUCTLSFILES)
-	${AR} $@ $(SSTRUCTMVFILES)
-	${AR} $@ $(STRUCTLSFILES)
-	${AR} $@ $(STRUCTMVFILES)
-	${AR} $@ $(UTILITIESFILES) $(BLASFILES) $(LAPACKFILES)
+	${AR} $@ $(FILES_HYPRE)
```

Comment:
This was originally broken into smaller pieces because some systems didn't like the long list (I can't remember the specifics, but might be able to do a bit of research to figure it out).  Is there a reason you are doing it with one line?